src/style-library/components/pages/RowLayoutScreen.scss

```
@@ -0,0 +1,54 @@
+}
+
+// Preset names are full words ("Muted Band"), not the short numeric labels the shared 4rem column
+// was sized for.
+.kadence-blocks-style-library__rowlayout-screen .kadence-blocks-style-library__list-row-label {
+	width: 15rem;
+}
+
+// The live slab's frame and checker. Proportions matter here rather than being decorative: a Row
+// Layout is always full-width and short relative to its width, and a corner radius reads very
+// differently at those proportions than on a square swatch, so the preview keeps a band's aspect.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)